doc-requirements.txt

```diff
@@ -1,2 +0,0 @@
-.[docs]
-
```

Comment:
This file is needed by ReadTheDocs. Currently building the docs fails over there. I am going to add it on master again.